src/legacy/core_plugins/dashboard_embeddable_container/public/index.scss

```
@@ -1,6 +1,6 @@
-
 @import 'src/legacy/ui/public/styles/styling_constants';
+@import 'src/legacy/core_plugins/embeddable_api/public/css/variables';
 
-// TODO: uncomment once the duplicate styles are removed from the dashboard app itself.
-// MUST STAY AT THE BOTTOM BECAUSE OF DARK THEME IMPORTS
-@import './embeddable/index';
+@import './css/dashboard_grid';
```

We'd still like to try to keep the SASS files next to their JS file counterparts so it's easier to find the right one. Can these not live in the `np_ready` folder even though their not used directly inside that folder but imported in this file instead?

We also would like to keep our `_index.scss` conventions if possible as well.

So for instance, can this `dashboard_grid` file live here?:
```suggestion
@import './np_ready/public/lib/embeddable/grid/index'; // This new index file would then import dashboard_grid
```

---

This would also make it easier to keep files where they are and then just moving the `index.scss` file to `np_ready` when the new platform can actually handle SASS compilation.

---

Good idea, thanks.

---

Done, see https://github.com/elastic/kibana/pull/41272/commits/8f568b22014c01e2d203a0ae0887448981ce7739